src/chocolatey/infrastructure/registration/SecurityProtocol.cs

```diff
@@ -34,9 +36,41 @@ public static void set_protocol(ChocolateyConfiguration config, bool provideWarn
                 // Framework 4.0. However if someone is running .NET 4.5 or 
                 // greater, they have in-place upgrades for System.dll, which
                 // will allow us to set these protocols directly.
-                const SecurityProtocolType tls11 = (SecurityProtocolType)TLS_1_1;
-                const SecurityProtocolType tls12 = (SecurityProtocolType)TLS_1_2;
-                ServicePointManager.SecurityProtocol = tls12 | tls11 | SecurityProtocolType.Tls | SecurityProtocolType.Ssl3;
+                
+                //Setting to system default to avoid carryover any default settings.
+                //System Default will be overwritten if any values added to it.
+                SecurityProtocolType allowedProtocolsList = SystemDefault;
+                foreach (string protocol in config.SecurityProtocols.AllowedSecurityProtocol.Split(','))

```

Comment:
You'll also want to remove empty entries here.

Reply:
Not sure what are you referring too ? Are you saying if AllowedSecurityProtocol is empty ?